source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+
+Home Assistant has a **discovery** feature, which scans for new devices as they’re added and adds them to your Home Assistant automatically.
+
+**Device trackers** are devices like cell phones that can move around.
+
+## Entities
+
+Because many devices do more than one thing, most things in Home Assistant use  **entities**. An entity is a single function or property of a device, like a single sensor, light, or switch.
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
+
+Some entities are useful for controlling things in your home:
+
+- **Switches** let you check if something is switched on or off?
+- **Sensors** can be a wide range of things that watch for changes in the environment, like motion sensors and temperature sensors
+- **Binary sensors** detect something that is either on or off
+- **[Scripts](https://www.home-assistant.io/docs/scripts/)** are special entities that run custom code when they’re triggered
+
+Other entities are there to respond to Home Assistant:
+
+- **Lights** can turn on and off, and sometimes dim and change color
+- **Covers** are things that can move up and down, like garage doors and smart blinds
+
+Some entities are part of a **domain**, or a built-in category of entities that are managed in a similar way. For example, there is a “lights” domain in Home Assistant.
+
+Entities can be grouped into **areas**, which roughly represent a room in your house. Common areas are “kitchen”, “office”, “basement” and “yard”.
+
+You can reference multiple entities at one time by putting them into a **[group](https://www.home-assistant.io/integrations/group/)**
+
+## Automation
+
+**[Automations](https://www.home-assistant.io/docs/automation/)** let you go father by telling Home Assistant to automatically do things without you having to hit a button yourself. For example, you could write an automation that says “turn on the lights when it gets dark if I’m home”.
+
+An automation has three main parts:
+
+- **[Triggers](https://www.home-assistant.io/docs/automation/trigger/)** - something that starts the automation, like the sun going down or a motion sensor noticing movement
+- **[Conditions](https://www.home-assistant.io/docs/automation/condition/)** - let you decide whether to complete the automation. For example: "only if someone is home".
+- **[Action](https://www.home-assistant.io/docs/automation/action/)** - what the automation does, like "turn on lights"
+
+**[Templates](https://www.home-assistant.io/docs/automation/templating/)** let advanced users use variables and logic in their automations
+
+## Events and state
+
+Entities in Home Assistant are controlled by sending **[events](https://www.home-assistant.io/docs/configuration/events/)**.
```

Comment:
Technically, under the hood, this is sort of true. However, is this really the experience from the user end? Is calling a service an event? I think it is better to set is as an action. For example, a service is an action.

The reasoning is that from the user end, we actually also handle events (e.g., in device automations). And you really don't want to create confusion around that.

Reply:
This section feels super wrong to me, but I just don't know how to fix it. What's an action? What's a service? I'll do my best but will probably need help. I'm also confused 🙂

Reply:
I just removed this sentence entirely